ambry-tools/src/main/java/com/github/ambry/account/MySqlAccountsDBTool.java

```diff
@@ -212,15 +214,19 @@ public void initialize() throws SQLException {
         zkFetchTimeMs - startTimeMs);
 
     AccountInfoMap accountInfoMap = new AccountInfoMap(new AccountServiceMetrics(new MetricRegistry()), accountMap);
+
+    // Populate Account and Container tables
     for (Account account : accountInfoMap.getAccounts()) {
-      accountTable.addAccount(account);
-      for (Container container : account.getAllContainers()) {
+      List<Container> containers = (List<Container>) account.getAllContainers();
+      for (Container container : containers) {
         containerTable.addContainer(account.getId(), container);
+        account.updateContainerMap(container, true);
```

Comment:
I don't see a reason to remove the containers one by one.  Simpler to empty out the map in one shot, or remove it from the json before inserting to DB (see above).

Reply:
Sure, made changes to remove containers from account while serializing to json as per your suggestion below.